src/translations/da_DK.json

```diff
@@ -252,7 +252,7 @@
   "ONBOARDING_DOWNLOAD_HEADLINE_PART_1": "",
   "LOGIN_APP_HEADING": "Log på",
   "LOGIN_APP_BTN_LABEL": "",
-  "LOGIN_APP_PAGE_TITLE": "",
+  "LOGIN_APP_PAGE_TITLE": "Sign in | Hedvig Insurance",
   "LOGIN_APP_INAVLID_SSN": "",
   "LOGIN_APP_GENERIC_ERROR": "Noget gik galt. Prøv igen eller kontakt os."
```

Comment:
Do you know why do we have these in Danish? The login screen won't be used for Denmark right? 🤔